seed/challenges/08-coding-interview-questions-and-take-home-assignments/coding-interview-data-structure-questions.json

```diff
@@ -893,50 +893,50 @@
       "hints": [],
       "type": "waypoint",
       "challengeType": 1,
-      "translations": {}
+      "translations": {}      
```

Comment:
Awesome thanks so much Mrugesh @raisedadead! I committed my changes and if there's anything else I need to do just let me know!